tflite/tflite-rtsp/docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: '3.9'  # You can specify the version that suits your needs
```

Comment:
Drop this whole line. `version` is not needed for anything since a long time now.